modern_treasury/modern_treasury.py

```diff
@@ -83,7 +83,7 @@ def _delete(self, url:str) -> dict:
 
     # Counter Parties
     def get_counter_parties(self):
-        return self._get(url=COUNTER_PARTIES_URL)
+        return self._get(url=COUNTER_PARTIES_URL)[1]
```

Comment:
I think Tuples and list calls make this code a brittle in my opinion.
What do you think for having a method that simply returns the response and you can extract the json from that?

```
def _get(self, url:str, params = None) -> Response:
    
    
 ...
def get_counter_parties(self):
    return self._get(url=COUNTER_PARTIES_URL).json()
```